okta/resource_app_oauth.go

```diff
@@ -235,6 +237,12 @@ func resourceAppOAuth() *schema.Resource {
 				Default:     true,
 				Description: "Do not display application icon to users",
 			},
+			"custom_profile_attributes": &schema.Schema{
+				Type:        schema.TypeString,
+				StateFunc:   normalizeDataJSON,
+				Optional:    true,
+				Description: "Custom JSON returned by GET requests to api/v1/apps/{appId}",
```

Comment:
Thanks for adding a description, one tweak, could we say `"Custom JSON that represents an OAuth application's profile"`

Reply:
Updated